_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig

```diff
@@ -471,7 +471,7 @@
         </div>
         <div class="large-19 columns">
             <ul class="inline-list">
-                <li><a class="copyright">Copyright © 2016 Red Hat Inc.</a></li>
+                <li><a class="copyright">Copyright © 2017 Red Hat Inc.</a></li>
```

Comment:
You could change this to something like what is shown at https://twig.symfony.com/doc/2.x/filters/date.html:
```    
{{ "now"|date("Y") }}
```
Then you wouldn't need to ever change it